packages/airnode-examples/README.md

```diff
@@ -16,12 +16,26 @@ but you may adapt the configuration to work for your target chain.
 
 ## Request-Response protocol (RRP)
 
-Currently, all of the examples facilitate the RRP protocol. The example RRP flow consists of two high level parts:
+Currently, all of the examples utilize the RRP protocol. The example RRP flow consists of two high level parts:
 
-1. Deploy an `AirnodeRrp` contract and a `RrpRequester` on a supported chain
-2. Deploy Airnode on a cloud provider (or run locally in a docker) and make a request using the deployed requester
+1. Deploy an `AirnodeRrp` contract and a `RrpRequester` contract on a supported chain
+2. Deploy Airnode on a cloud provider, or run locally in a docker container, and make a request using the deployed
+   requester
 
-If you would like to know more about RRP, read the [API3 docs](https://docs.api3.org/airnode/v0.2/concepts/).
+If you would like to know more about RRP, read the [API3 docs](https://docs.api3.org/airnode/v0.3/concepts/).
+
+## Available examples
+
+We call examples "integrations" because they are integrated with some API. With each integration you are offered a
+choice of how you want to run the Airnode and which network to use, as mentioned above. The integrations have been
+designed to highlight various Airnode functionality and use cases, from simple price requests to more complex
+authenticated requests encoding multiple reserved parameters. The following list orders integrations by increasing
```

Comment:
Order by complexity is a subjective thing and will be hard to follow as the list grows in size. I think we can just use the folder names in aplhabetical order.

Reply:
These folder names would ideally be links.

Reply:
And ideally there would be a unit test that would verify that all of them are listed.

Reply:
> And ideally there would be a unit test that would verify that all of them are listed.

This is probably an overkill since this will be migrated in the docs.

Reply:
Thanks for the feedback. I made the requested updates:
- Changed to alphabetical
- Added links
- Mentioned integration addition instructions